### PR TITLE
Prepare 1.0.1 release

### PR DESCRIPTION
**Pull request by justinfagnani**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,14 +10,27 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
...
+<!-- ## Unreleased -->
 <!-- ### Changed -->
 <!-- ### Added -->
 <!-- ### Removed -->
+<!-- ### Fixed -->
+
+## [1.0.1] - 2019-05-20
+
+### Changed
+* Many small performance enhancement.
```
Comment (jridgewell) [inline CHANGELOG.md]: Nit: enhancements

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,14 +10,27 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
...
+* Private names are now named with a `__` prefix  ([#859](https://github.com/Polymer/lit-html/issues/859)).
+
+### Added
+* Setup continuous benachmarking with Tachometer ([#887](https://github.com/Polymer/lit-html/issues/887)).
```
Comment (kenchris) [inline CHANGELOG.md]: Benchmarking*

**Comment by justinfagnani** [comment]
@sorvell I ran a dev release through lit-element tests: https://github.com/Polymer/lit-element/pull/679

All green.

**Comment by web-padawan** [comment]
@justinfagnani out of curiosity, was it planned to release 1.0.1 or 1.1.0 ?

**Comment by justinfagnani** [comment]
@web-padawan uh... it was supposed to be 1.0.1, but I typed `np minor` instead of `np patch`, and I don't think there was any confirmation prompt before it published. So... oops :/

**Comment by web-padawan** [comment]
@justinfagnani that's that I thought. "Manual work is a bug".

WDYT about [semantic release](https://github.com/semantic-release/semantic-release) or any other tool to release in CI after merging to master, based on conventional commits?

It would also would generally improve the situation so that fixes are immediately delivered. There has been a number of commits in master unreleased for a while.

I do understand the need to also run `lit-element` tests but that could likely be done in CI using `npm link` as well, right?

**Pull request merged**